metals-bench/src/main/scala/bench/ClasspathIndexingBench.scala

```diff
@@ -31,7 +32,10 @@ class ClasspathIndexingBench {
   @BenchmarkMode(Array(Mode.SingleShotTime))
   @OutputTimeUnit(TimeUnit.MILLISECONDS)
   def run(): Unit = {
-    ClasspathSearch.fromClasspath(classpath)
+    ClasspathSearch.fromClasspath(
+      classpath,
+      new ExcludedPackagesHandler(None).isExcludedPackage
```

Comment:
maybe we could make `pkgsToExclude` default value 'None' and drop providing it in those test cases.

Reply:
Sure, I've gone ahead and changed this.